src/js/tabs/firmware_flasher.js

```diff
@@ -1023,7 +1023,9 @@ TABS.firmware_flasher.initialize = function (callback) {
         portPickerElement.change(function () {
             if ($('option:selected', this).data().isDFU && !GUI.connect_lock) {
```

Comment:
```suggestion
            if (!GUI.connect_lock) {
              if ($('option:selected', this).data().isDFU) {
                exitDfuElement.removeClass('disabled');
              } else {
                exitDfuElement.addClass('disabled'):
                $("a.load_remote_file").removeClass('disabled');
                $("a.load_file").removeClass('disabled');
              }
            }
```

Reply:
It probably does not make a big parctical difference, but in general it probably makes sense to prevent changes to the button state while flashing is in progress. But maybe `GUI.connect_lock` needs a different name, now that it is doing more than just disabling the connect button.

Reply:
`GUI.flash_in_progress`

Reply:
Good idea to rename this element, but does `GUI.connect_lock` is only present  in Firmware flasher tab? or we are using in other places?

Reply:
@Asizon: Obviously it will have to be renamed whereever it is used. And where it is used, and what for, should be considered in the choice of an appropriate name.

Reply:
Perfect @mikeller , maybe best to make these changes in another PR?

Reply:
@Asizon: Can do - the important part is that it is done, as this pull request changes the use of this property, making its name inappropriate.